global.js

```diff
@@ -158,7 +174,12 @@ teamKeeper = function(player, leaving) {
 
 if (!ts.eventsRegistered) {
     Events.on(EventType.PlayerConnect, cons(e => kickpirated(e.player)))
-
+    
+    Events.on(EventType.PlayerChatEvent, cons(e => {
+        if (e.message.toLowerCase().includes("silicone")) silicone(e.player)
+        if (e.message.toLowerCase().includes("polies")) silicone(e.player)
```

Comment:
```suggestion
        if (e.message.toLowerCase().includes("polies")) polies(e.player)
```
i think you meant `polies` here